src/handbook/company/communication.md

```diff
@@ -75,12 +75,15 @@ below so you get to decide if you want more channels or not.
 
 #### Channel name structure
 
-When a channel is created for projects; work that's scoped to a certain topic
-that's going to be completed at some time in the future, it might be valuable to
-collect all internal communications into one Slack channel. Please prefix the
-channel name with `proj-` to make grouping easier.
+To make it easier to find your way around in Slack, it's 
```

Comment:
Not sure where you planned to go with this sentence, but here's a proposed second half...

```suggestion
To make it easier to find your way around in Slack, we have some rules for naming channels. This will help give everything some context and implicit sense of purpose.
```